doc/source/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.rst

```diff
@@ -0,0 +1,95 @@
+======================================
+2D Flow around a cylinder
+======================================
+
+In this example, a flow is passing across a fixed cylinder. The velocity profile of the flow is simulated. The parameter file used is ``cylinder.prm``. This example can be solved using the ``gls_navier_stokes_2d`` solver.
+
+The following schematic describes the simulation.
+
+.. image:: images/geometry_bc.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry_and_bc
+
+
+* bc = 0 (No slip boundary condition)
+* bc = 1 (u = 1; flow in the x-direction)
+* bc = 2 (Slip boundary condition)
+
+Initial conditions
+------------------
+The initial condition implemented describes the movement of the flow in the x-direction.
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 1; 0; 0
+        end
+    end
+
+The ``subsection uvwp`` allows the description of a vector-valued function representing the velocity in the x-direction where the individual components of the velocity are separated by semicolons in the ``set Function expression``.
+
+Mesh
+----
+For the mesh, the ``gmsh`` option was chosen as which requires the input of a ``.msh`` file generated by `Gmsh <https://gmsh.info/#Download>`_.
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 = gmsh
+        set file name            = cylinder_structured.msh
+    end
+
+The ``.msh`` file can be found in the same folder where the parameter file was obtained. The ``set file name`` includes the path of the file to be used, however since in this case both the parameter and the mesh files are located in the same folder, no path needs to be identified.
```

Comment:
```suggestion
The ``.msh`` file can be found in the same folder where the parameter file was obtained. The ``set file name`` includes the path of the file to be used, however since in this case both the parameter and the mesh files are located in the same folder, no path needs to be identified. The ``.geo`` used to generate the gmsh mesh is also provided in this folder as a reference.
```